### PR TITLE
[home] implement changes based on dogfooding feedback

### PR DESCRIPTION
**Pull request by FiberJW**
# Why

Last Friday, we had a dogfooding session where I collected a lot of good feedback on how to make this app better. 

# How

I made the commits based on each feedback item, so please read the commit list to see what I changed. The biggest changes visually are the new flatlist styles, which look like this: 

Screen Shot 2022-04-03 at 13 19 18

Screen Shot 2022-04-03 at 14 10 27

# Test Plan

Run this app on iOS and Android and make sure it feels good and works as expected

### CODE REVIEW

**Comment by ajsmth** [comment]
its looking good - should I be implementing similar design changes in dev-launcher?

**Comment by FiberJW** [comment]
> its looking good - should I be implementing similar design changes in dev-launcher?

My general feedback for the lists is that the whole page should scroll, not just the nested container view with the rounded corners. If your lists scroll like that/have the scroll bar on the absolute edge of the screen, you could keep it, but I found it hard to implement the list designs and have that consistent page-scrolling behavior.

TLDR: probably yes

**Review by ajsmth — APPROVED** [review]
